stories/button.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { checkA11y } from 'storybook-addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import { IconAddMediumOutline, IconMagicMediumOutline } from '@teamleader/ui-icons';
+import { Button, ButtonGroup } from '../components/button';
+
+storiesOf('Buttons', module)
+  .addDecorator((story, context) => withInfo('')(story)(context))
+  .addDecorator(checkA11y)
+  .add('with text', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked button 1')}>button 1</Button>
+      <Button onClick={action('clicked button 2')}>button 2</Button>
+    </ButtonGroup>
+  ))
+  .add('with icon', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked button 1')}>
+        <IconAddMediumOutline />
+      </Button>
+      <Button onClick={action('clicked button 1')}>
+        <IconMagicMediumOutline />
+      </Button>
+    </ButtonGroup>
+  ))
+  .add('with text and icon', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked button 1')}>
+        <IconAddMediumOutline />
+        button
+      </Button>
+      <Button onClick={action('clicked button 2')}>
+        button
+        <IconAddMediumOutline />
+      </Button>
+    </ButtonGroup>
+  ))
+  .add('segmented', () => (
+    <ButtonGroup segmented>
+      <Button onClick={action('clicked')}>
+        <IconAddMediumOutline />
+      </Button>
+      <Button onClick={action('clicked button 1')}>button 1</Button>
+      <Button onClick={action('clicked button 2')}>button 2</Button>
+    </ButtonGroup>
+  ))
+  .add('disabled', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked button 1')} disabled>
+        <IconAddMediumOutline />
+      </Button>
+      <Button onClick={action('clicked button 2')} disabled>
+        button
+      </Button>
+    </ButtonGroup>
+  ))
+  .add('processing', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked button 1')} processing>
+        <IconAddMediumOutline />
+      </Button>
+      <Button onClick={action('clicked button 2')} processing>
+        button
+      </Button>
+    </ButtonGroup>
+  ))
+  .add('themes', () => (
+    <ButtonGroup>
+      <Button onClick={action('clicked primary button')} level="primary">
+        Primary button
+      </Button>
+      <Button onClick={action('clicked secondary button')} level="secondary">
+        Secondary button
+      </Button>
+      <Button onClick={action('clicked ouline button')} level="outline">
+        Outline button
+      </Button>
+      <Button onClick={action('clicked destructive button')} level="destructive">
+        Destructive button
+      </Button>
+    </ButtonGroup>
+  ))
+  .add('inverse', () => (
```

Comment:
Can you change the background color of whatever is holding the `<Button>`s? There's no visual difference between a regular button and an inverse button now?

Reply:
Done.

<img width="742" alt="screen shot 2017-10-11 at 11 02 25" src="https://user-images.githubusercontent.com/1453781/31431345-b7c9c972-ae73-11e7-97cc-a47c0a54f600.png">